libs/cardano-ledger-conformance/src/Test/Cardano/Ledger/Conformance/SpecTranslate/Conway/Base.hs

```diff
@@ -602,7 +602,13 @@ instance SpecTranslate ctx RewardAccount where
 instance SpecTranslate ctx PoolParams where
   type SpecRep PoolParams = Agda.PoolParams
 
-  toSpecRep PoolParams {..} = Agda.PoolParams <$> toSpecRep (KeyHashObj ppId)
+  toSpecRep PoolParams {..} =
+    Agda.PoolParams
+      <$> toSpecRep ppOwners
+      <*> toSpecRep ppCost
+      <*> toSpecRep ppMargin
+      <*> toSpecRep ppPledge
+      <*> toSpecRep (KeyHashObj ppId)
```

Comment:
I'm unsure about the last line:
```haskell
 <*> toSpecRep (KeyHashObj ppId)
 ```
 Previously the Agda definition of `PoolParams` had only one field `rewardAddr : Credential` which is what `ppId` was mapped to.

This field has been [replaced](https://github.com/IntersectMBO/formal-ledger-specifications/commit/7fc02421eae3d926917feccb141f8149d481466c) by `rewardAccount : Credential` in the Agda side so I'm unsure if we should use `ppId` as before or with something that involves `ppRewardAccount`.